src/passes/pass.cpp

```diff
@@ -620,6 +623,7 @@ void PassRunner::addDefaultGlobalOptimizationPrePasses() {
     }
     addIfNoDWARFIssues("remove-unused-module-elements");
     if (options.closedWorld) {
+      addIfNoDWARFIssues("remove-unused-types");
```

Comment:
Can we skip adding this pass in nominal mode?

Reply:
Sure, but after we delete nominal mode it won't have made a difference.

Reply:
Right, fair point.